frontend/src/views/ProjectDashboard.vue

```diff
@@ -4,7 +4,7 @@
       <AdvancedSearch
         v-model:params="state.params"
         :autofocus="false"
-        :placeholder="$t('common.filter-by-name')"
+        :placeholder="$t('common.filter-by-name-or-label')"
```

Comment:
We already support the filter by database labels now

<img width="934" height="264" alt="Image" src="https://github.com/user-attachments/assets/b56c4500-ce02-4d09-ac89-c7442ca7f769" />

I want the UX to be unified, and it's better to follow the current UX. Maybe need to rename the `database-label` to a common name, and keep the `{key}:{value}` pair format in the `AdvancedSearch`. Then we can add another field named `lables: {key: string; value: string}[]` and build the filter